### PR TITLE
BarcodeScanner BlackBerry 10 support

### PR DESCRIPTION
**Pull request by umcsdon**
Related to https://github.com/blackberry/WebWorks-Community-APIs/issues/386

### CODE REVIEW

**Comment by umcsdon** [comment]
## Test Project

### index.html
```html
<html>
    <head>
        <meta charset="utf-8" />
        <meta name="format-detection" content="telephone=no" />
        <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi" />
        <link rel="stylesheet" type="text/css" href="css/index.css" />
        <title>Blackberry 10 - Cordova - Barcode Scanner</title>
    </head>
    <body>
        <div id="appContainer" class="app">
            <h1>Barcode Scanner <br />Sample App</h1>
            <button id="barcodeScanner_start">Start Read</button>

            <br />
            <br />

            <div id="results"></div>
        </div>
        <script type="text/javascript" src="cordova.js"></script>
        <script type="text/javascript" src="js/index.js"></script>
        <script type="text/javascript">
            app.initialize();
        </script>
    </body>
</html>
```

### www/js/index.js
```javascript
var app = {
	barcodeScanner: null,
	results: null,
	appContainer: null,

	// Application Constructor
	initialize: function() {
		this.bindEvents();
	},
	// Bind Event Listeners
	//
	// Bind any events that are required on startup. Common events are:
	// 'load', 'deviceready', 'offline', and 'online'.
	bindEvents: function() {
		document.addEventListener('deviceready', app.onDeviceReady, false);
	},
	// deviceready Event Handler
	//
	// The scope of 'this' is the event. In order to call the 'receivedEvent'
	// function, we must explicity call 'app.receivedEvent(...);'
	onDeviceReady: function() {
		results = document.getElementById("results")

		app.barcodeScanner = cordova.plugins.barcodeScanner
		appContainer = document.getElementById("appContainer")
		document.getElementById('barcodeScanner_start').addEventListener('click', app.startRead, false)
	},

	startRead: function() {
		app.barcodeScanner.scan(app.codeFound, app.errorFound)
	},

	errorFound: function(data){
		// console.log("Error : "+ data.error + " description : "+ data.description);
		console.log("Error : " + data);
	},

	writeOut: function(message) {
		results.innerText = results.innerText + message;
		results.appendChild(document.createElement('br'));
	},

	codeFound: function(data) {
		app.writeOut(
			"We got a barcode\n" +
			"Result: " + data.text + "\n" +
			"Format: " + data.format + "\n" +
			"Cancelled: " + data.cancelled
		);
	}
};
```

File: README.md [hunk 1]
```diff
@@ -129,10 +141,13 @@ Windows 8 implenemtation currently doesn't support encode functionality.
...
 
-## Thanks on Github ##
+## BlackBerry 10 quirks
+BlackBerry 10 implenematation currently doesn't support encode functionality.
 
-So many -- check out the original [iOS](https://github.com/phonegap/phonegap-plugins/tree/master/iOS/BarcodeScanner) and [Android](https://github.com/phonegap/phonegap-plugins/tree/master/Android/BarcodeScanner) repos.
+## Thanks on Github ##
 
+So many -- check out the original [iOS](https://github.com/phonegap/phonegap-plugins/tree/master/iOS/BarcodeScanner),  [Android](https://github.com/phonegap/phonegap-plugins/tree/master/Android/BarcodeScanner) and 
+[BlackBerry 10](https://github.com/umcsdon/WebWorks-Community-APIs/tree/master/BB10-Cordova/BarcodeScanner) repos.
```
Comment (timwindsor) [inline README.md]: Should probably link back to the main blackberry repo instead of your own, in case you later delete your fork.

**Comment by timwindsor** [comment]
Otherwise I think we're ready to merge this in

**Comment by umcsdon** [comment]
Fixed

**Pull request merged**